backend/src/models/renter.ts

```diff
@@ -0,0 +1,17 @@
+import { InferSchemaType, Schema, model } from "mongoose";
+
+const renterSchema = new Schema(
+  {
+    firstName: { type: String, required: true },
+    lastName: { type: String, required: true },
+    contactInfo: { type: String, required: true },
+    program: { type: String, required: false },
+  },
```

Comment:
Sorry, this is totally my fault! I forgot to update the schema to reflect the design updates (referring to [this design](https://www.figma.com/file/06vEDN5gtlHlC5hxNXMb4g/USHS-Designs?node-id=1186%3A14552&mode=dev)). The schema should up to date now. Do you mind updating to match? 🙏🏼

Reply:
Updated!